src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -311,6 +311,15 @@ void RegisterAppInterfaceRequest::Run() {
   SendRegisterAppInterfaceResponseToMobile();
 }
 
+void FillVRRelatedFields(smart_objects::SmartObject& response_params,
```

Comment:
@LuxoftAKutsan what is this? If this is an anonymous function please put it to anonymous namespace.